foundational-cxx-support-matrix.md

```diff
@@ -19,10 +19,10 @@
 | RockyLinux      | >= 9                   | 2024-07-01   | 2027-06-01 |
 | Windows Server  | >= 2022                | 2024-01-22   | 2026-10-13 |
 | Windows Client  | >= 10                  | 2022-07-01   | 2025-11-01 |
-| macOS           | >= 11 (Big Sur)        | 2024-09-17   | 2025-07-30 |
-| iOS             | >= 15                  | 2024-01-24   | 2025-07-30 |
+| macOS           | >= 14 (Big Sur)        | 2025-09-15   | 2026-09-01 |
+| iOS             | >= 18                  | 2025-09-15   | 2026-09-01 |
```

Comment:
This is a pretty big jump (~3 years worth of releases), and around a [quarter](https://discuss.python.org/t/moving-packaging-and-installers-to-macos-10-13-as-a-minimum/31907/9) of macOS users (as measured through PyPI download metadata) are still on <14.

A more expected increment would be to drop one release per year (just as Apple adds a new release a year).